Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "multi-party-ecdsa"
-version = "0.3.8"
+version = "0.2.9"
```

Comment:
Why not 0.3.0?
It's a breaking change afaik... return type in an API function is different